LICENSE.md

```diff
@@ -1,3 +1,7 @@
-Copyright (C) 2022 Micah Hoffman
+Copyright (C) 2022 Micah Hoffman, Dan Cardin

```

Comment:
I'd like to remain the sole copyright-holder for the project. 
Can you add a line under my copyright specifying you hold copyright on the script?

Reply:
This is done.